src/doc/rgw/user-accounts.md

```diff
@@ -55,6 +55,19 @@ this requires the account metadata to also manage the index of all buckets under
 we should be able to use the same cls_user calls for `{accountid}.buckets` that we already use for `{userid}.buckets`. so if we solve this scaling problem for users, we could apply the same solution to accounts
```

Comment:
this scaling discussion drops down to details of the RADOS implementation, but as of today, I think we need to also consider alternate stores;  are there any aspects of this design that aren't trivially covered by zipper abstractions and compatible with dbstore?

Reply:
>  are there any aspects of this design that aren't trivially covered by zipper abstractions and compatible with dbstore?

nope! sal has interfaces to read/write the quota limits on buckets/users, and a `Bucket::check_quota()` that's responsible for quota enforcement. backends can implement the quota accounting for that however they choose

this design goes into details of the rados backend because it's the only one that implements quota